### ISSUE TITLE
Add string interpolation to all TICKscript strings

### ISSUE THREAD

**Issue by nathanielc** [influxdata/kapacitor#1288]
Its a common enough practice to use vars as part of string literals in TICKscript we should have native support for string interpolation, therefore removing the need to use the verbose string concatenation methods.

I am thinking something along the lines of:

```
var field = 'count'

var db = 'mydb'
var rp = 'rp'
var m  = 'm'
var actual = batch
    |query('SELECT ${field} FROM "${db}"."${rp}"."${m}" ')
    //...
```

**Comment by bbinet** [comment]
Yes, this would be really useful to have.
Any ETA?

**Comment by lblt** [comment]
How can I do if I have multiple fields/tags? Do I have to declare them as
```
var field1
var field2
var tag1
...
```
and then to iterate the 
```
'SELECT "' + field1 + '","' + field2 + ...
```
? Seems complicated...

**Comment by nathanielc** [comment]
@lblt Yes, you are correct. And the current state is complicated hence this open issue to improve.

**Comment by lblt** [comment]
Hi @nathanielc thank you for your answer!

**Comment by chamini2** [comment]
Related to #1109

**Comment by sinistram** [comment]
+ to the feature. It is very painful to support scripts without string interpolation
